dali/core/nvtx.cc

```diff
@@ -56,6 +69,18 @@ DLL_PUBLIC DomainTimeRange::~DomainTimeRange() {
   DomainTimeRangeImpl::GetInstance().Stop();
 }
 
+
+DLL_PUBLIC void SetThreadName(const char *name) {
+  nvtxNameOsThreadA(syscall(SYS_gettid), name);
```

Comment:
That's a heavy hammer. Include `sys/types.h` and:
```suggestion
  nvtxNameOsThreadA(gettid(), name);
```

Reply:
Done